carma_wm/include/carma_wm/WMTestLibForGuidance.h

```diff
@@ -0,0 +1,458 @@
+  all_lanelets.push_back(getLanelet(1202, ls02,ls12,lanelet::AttributeValueString::Solid, lanelet::AttributeValueString::Dashed));
+  all_lanelets.push_back(getLanelet(1203, ls03,ls13,lanelet::AttributeValueString::Solid, lanelet::AttributeValueString::Dashed));
+
+  all_lanelets.push_back(getLanelet(1210, ls10,ls20,lanelet::AttributeValueString::Dashed, lanelet::AttributeValueString::Dashed));
+  all_lanelets.push_back(getLanelet(1211, ls11,ls21,lanelet::AttributeValueString::Dashed, lanelet::AttributeValueString::Dashed));
+  all_lanelets.push_back(getLanelet(1212, ls12,ls22,lanelet::AttributeValueString::Dashed, lanelet::AttributeValueString::Dashed));
+  all_lanelets.push_back(getLanelet(1213, ls13,ls23,lanelet::AttributeValueString::Dashed, lanelet::AttributeValueString::Dashed));
+  
+  all_lanelets.push_back(getLanelet(1220, ls20,ls30,lanelet::AttributeValueString::Dashed, lanelet::AttributeValueString::Solid));
+  all_lanelets.push_back(getLanelet(1221, ls21,ls31,lanelet::AttributeValueString::Dashed, lanelet::AttributeValueString::Solid));
+  all_lanelets.push_back(getLanelet(1222, ls22,ls32,lanelet::AttributeValueString::Dashed, lanelet::AttributeValueString::Solid));
+  all_lanelets.push_back(getLanelet(1223, ls23,ls33,lanelet::AttributeValueString::Dashed, lanelet::AttributeValueString::Solid));
+  
+  // Create basic map and verify that the map and routing graph can be build, but the route remains false	
+  lanelet::LaneletMapPtr map = lanelet::utils::createMap(all_lanelets, {});
+  return map;
+}
+
+/**
+ * \brief adds a roadway object at the specified cartesian coords
+ * \param x coord
+ * \param y coord
+ * \param cmw CARMAWorldModel shared ptr
+ * \param pred_coords vector of std::pair(x,y) predicted coords with 1s interval in the future
+ * \param width width of roadway object, default 3 meters
+ * \param length length of roadway object, default 3 meters
+ * NOTE: x,y is the center of your object
+ */
+inline void addObstacle(double x, double y, std::shared_ptr<carma_wm::CARMAWorldModel> cmw, std::vector<std::pair<double,double>> pred_coords = {}, double width = 3, double length = 3)
+{
+    cav_msgs::RoadwayObstacle rwo;	
+
+    tf2::Quaternion tf_orientation;	
+    tf_orientation.setRPY(0, 0, 1.5708); // 90 deg
+
+    rwo.object.pose.pose.position.x = x;	
+    rwo.object.pose.pose.position.y = y;	
+    rwo.object.pose.pose.position.z = 0;	
+
+    rwo.object.pose.pose.orientation.x = tf_orientation.getX();	
+    rwo.object.pose.pose.orientation.y = tf_orientation.getY();	
+    rwo.object.pose.pose.orientation.z = tf_orientation.getZ();	
+    rwo.object.pose.pose.orientation.w = tf_orientation.getW();	
+
+    rwo.object.size.x = width;	
+    rwo.object.size.y = length;	
+    rwo.object.size.z = 1;	
+
+    int time_stamp = 0;
+    std::vector<cav_msgs::PredictedState> pred_states;
+    for (auto pred : pred_coords)
+    {
+        cav_msgs::PredictedState ps;
+        time_stamp += 1000;
```

Comment:
Change this to
10000000

This is 100ms in ns. Currently our predictions are in 100ms increments so this will more accurately reflect the real system.

Reply:
Or maybe let the user specify the timestep size